src/main/java/com/nkh/ECommerceShop/controller/OrdersController.java

```diff
@@ -0,0 +1,30 @@
+package com.nkh.ECommerceShop.controller;
+
+import com.nkh.ECommerceShop.model.order.Order;
+import com.nkh.ECommerceShop.repository.OrdersRepository;
+import jakarta.validation.OverridesAttribute;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/orders")
+public class OrdersController {
+    private final OrdersRepository ordersRepository;
+
+    @Autowired
+    public OrdersController(OrdersRepository ordersRepository) {
+        this.ordersRepository = ordersRepository;
+    }
+
+    @GetMapping
+    @PreAuthorize("hasAnyRole('USER')")
+    public ResponseEntity<List<Order>> getAllOrders(){
+        return ResponseEntity.ok(ordersRepository.findAll());
```

Comment:
what if `.findAll()` throws an `SQLException`. As much as I appreciate brevity in code and doing multiple things within